selftests/unit/test_data_structures.py

```diff
@@ -29,6 +29,16 @@ def test_geometric_mean(self):
         self.assertEqual(data_structures.geometric_mean(xrange(1, 180)),
                          67.1555819421869)
 
+    def test_compare_matrices(self):
+        """
+        Verify the correct value is produced when comparing matrices.
+        """
+        matrix1 = [[4, 51.7, 60], [6, 0, 0]]
```

Comment:
Replacing the existing one should be IMO sufficient and yes, the comment was meant for inclusion (because usually you don't have matrices where some rows start with header and other with value...)